index.html

```diff
@@ -1179,29 +1170,27 @@
 tracking, a form of [=cross-context recognition=], but it can also be used for other
 inappropriate information flows.
 
-## Data minimization {#data-minimization}
+## Data Minimization {#data-minimization}
 
 <div class="practice">
-<span class="practicelab">Sites, user agents and other parties should minimize the amount of
+<span class="practicelab">Sites, user agents, and other parties should minimize the amount of
 <a>personal data</a> they transfer between parties on the Web.</span>
 </div>
 
-Data minimization limits the risks of data being disclosed or misused and also makes it possible to
+Data minimization limits the risks of data being disclosed or misused, and also makes it possible to
```

Comment:
```suggestion
Data minimization limits the risks of data being disclosed or misused, and it also makes it possible to
```

And maybe avoid the passive voice here:
> ... , and it also helps user agents more meaningfully explain the decisions their users need to make.

Reply:
Unpassived it.